io/plugins/eu.esdihumboldt.hale.io.csv.ui/src/eu/esdihumboldt/hale/io/csv/ui/TypeSelectionPage.java

```diff
@@ -48,8 +49,10 @@
 public class TypeSelectionPage extends InstanceReaderConfigurationPage implements CSVConstants {
 
 	private TypeDefinitionSelector sel;
-	private Button button;
-	private Label label;
+	private Text text;
+	private Spinner spinner;
```

Comment:
Maybe name the variable in a way that it is more clear which setting this component is associated to?